cli/src/helpers/installDependencies.ts

```diff
@@ -43,8 +43,12 @@ const runInstallCommand = async (
               : text;
           }
         });
-        pnpmSubprocess.on("error", (e) => rej(e));
-        pnpmSubprocess.on("close", () => res());
+        pnpmSubprocess.on("error", (e) => {
+          rej(e);
+        });
+        pnpmSubprocess.on("close", () => {
+          res();
+        });
```

Comment:
https://v6--typescript-eslint.netlify.app/rules/no-confusing-void-expression flags cases around `void` such this: a function that returns a value being used in a place that expects it to return `void`. It's in the _`stylistic-type-checked`_ config in v6, after not being in any recommended configs in v5. I worry it's too opinionated and will annoy people. Thoughts?

Reply:
hmm i do see some sort of value for it but i'd probably disable it for most of my projects.

definetely on the more opinionated side and i think, as far as i'm concerned, it's on the wrong side of the line at least for me.

Reply:
Makes sense - I'll disable it in this PR & undo the changes made for it.